KinSDK/KinSDK/Core/blockchain/types/BaseTransaction.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  BaseTransaction.swift
+//  KinSDK
+//
+//  Created by Corey Werner on 01/07/2019.
+//  Copyright © 2019 Kin Foundation. All rights reserved.
+//
+
+import Foundation
+
+public class BaseTransaction {
+    var transaction: Transaction
+
+    init(wrapping transaction: Transaction) {
+        self.transaction = transaction
+    }
+
+    public var fee: Quark {
+        return transaction.fee
+    }
+
+    public func hash(networkId: Network.Id) throws -> Data {
```

Comment:
what's `Data`? hash should return plain String

Reply:
This is how the code was when I got it. I don't want to make too many changes at once. I'll make a note of this and we can come back to it.